File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Prices/Price.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Prices;

use Ohtap\ECommerce\DomainModels\Objects\Currencies\ICurrencyDataList;

class Price  implements IPrice
{


    public  ICurrencyDataList $Currencies;
    public string $Description;
    public int $State;
    public int $OrderNo;
    public string $ID;
    public string $Name;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Prices/Price.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Currencies;
using Ohtap.ECommerce.Domain.Models.Objects.Prices;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Prices
{
    public class Price : PriceBasic, IPrice
    {
        public ICurrencyDataList Currencies { set; get; }
    }
}
